### ISSUE TITLE
Multiple PNG plots can spill out of collapsed output cell

### ISSUE THREAD

**Issue by jiahao** [JuliaLang/IJulia.jl#285]
A cell that generates lots of PNG plots can produce an output cell in IPython 3.0 that, when collapsed, spills out of the cell boundaries.

```jl
using Color
using Compose
for i=1:3
    draw(PNG(400px, 400px), compose(context(), circle(0.5, 0.5, 100rand()), fill(RGB(rand(3)...))))
end
```

![screen shot 2015-03-05 at 11 25 00 pm](https://cloud.githubusercontent.com/assets/1732/6520206/07565686-c38f-11e4-8b9e-ef28c1781cda.png)

This problem is not seen with Compose's SVG rendering instead of PNG.

Julia 0.4-dev on IPython 3.0, Compose.jl and Color.jl checked out on master.

**Comment by stevengj** [comment]
It sounds like this should be an IPython bug, not an IJulia bug.

**Comment by minrk** [comment]
@jiahao can you share the notebook? I can't seem to run the code, but I'd like to see the HTML produced. It may well be an IPython issue.

**Comment by jiahao** [comment]
@minrk here it is: https://gist.github.com/jiahao/67fd5cccdf431c5674bc

**Comment by minrk** [comment]
@jiahao what browser, browser version and OS? I'm not seeing the issue in your notebook on Chrome 41, FF 36, or Safari 8 on OS X 10.10.2.

**Comment by jiahao** [comment]
@minrk Here is another example: https://gist.github.com/jiahao/cda9fc190730d42c5d0f

I see this on Chrome 41.0.2272.89 (64-bit) on OSX 10.10.2. I was not able to reproduce on Safari.

As we had discussed offline, the scrolling issue persists after reloading the notebook, and also after turning off my custom CSS.

**Comment by minrk** [comment]
@jiahao thanks. I believe this should be fixed by ipython/ipython#8082

**Comment by stevengj** [comment]
Thanks @minrk, closing this issue; @jiahao can re-open if he finds it is not fixed.
